File: Service_eHospital_DongNai_A/Implement/me_DM_MACAddressService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class me_DM_MACAddressService : BaseService<me_DM_MACAddress, Ime_DM_MACAddressRepository>
    , Ime_DM_MACAddressService
    {
    private readonly Ime_DM_MACAddressRepository _me_DM_MACAddressRepository;
    public me_DM_MACAddressService(Ime_DM_MACAddressRepository me_DM_MACAddressRepository) : base(me_DM_MACAddressRepository)
    {
    _me_DM_MACAddressRepository = me_DM_MACAddressRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/me_DM_MACAddress.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { me_DM_MACAddress } from './me_DM_MACAddress.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class me_DM_MACAddressService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'MAC_id', 'MacAddress', 'MoTa', 'NgayTao', 'NgayCapNhat'];

    List: me_DM_MACAddress[] | undefined;
    ListFilter: me_DM_MACAddress[] | undefined;
    FormData!: me_DM_MACAddress;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "me_DM_MACAddress";
    }
}

